src/js/providers/flash.js

```diff
@@ -348,18 +344,10 @@ define([
                     }
                     _container = null;
                     _item = null;
-                    _eventDispatcher.resetEventListeners();
-                    _eventDispatcher = null;
+                    this.off();
                 }
         });
 
-        // Overwrite the event dispatchers to block on certain occasions
-        this.sendEvent = function() {
-            if (!_attached) {
```

Comment:
The functionality of this `if (!_attached) {` statement is important, and it looks like it's been removed.